resources/public/api/conf/2.0/schemas/historic_uk_non_fhl_period_summary_retrieve/response.json

```diff
@@ -0,0 +1,161 @@
+          "examples": "5000.99"
+        },
+        "repairsAndMaintenance": {
+          "type": "number",
+          "description": "Property repairs and maintenance.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "financialCosts": {
+          "type": "number",
+          "description": "Loan interest and other financial costs.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "professionalFees": {
+          "type": "number",
+          "description": "Legal, management and other professional fees.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "costOfServices": {
+          "type": "number",
+          "description": "Cost of services provided, including wages.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "other": {
+          "type": "number",
+          "description": "Other allowable property expenses.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "consolidatedExpenses": {
+          "type": "number",
+          "description": "The sum of all expenses for the specified period.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "travelCosts": {
+          "type": "number",
+          "description": "Car, van and travel costs incurred in running a property business.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "residentialFinancialCostsCarriedForward": {
+          "type": "number",
+          "description": "Amount of residential financial costs carried forward.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "5000.99"
+        },
+        "residentialFinancialCost": {
+          "type": "number",
+          "description": "Captures residential financial cost that can be deductible from rental income (tax relief). The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+          "examples": "1000.45"
+        },
+        "rentARoom": {
+          "type": "object",
+          "description": "Object holding the amount claimed for the period",
+          "properties": {
+            "amountClaimed": {
+              "type": "number",
+              "description": "The amount of UK Furnished Holiday Lettings rent claimed.\nThe value must be between 0 and 99999999999.99 up to 2 decimal places.",
+              "examples": "5000.99"
+            }
+          }
+        }
+      }
+    },
+    "links": {
+      "type": "array",
+      "items": [
+        {
+          "type": "object",
```

Comment:
None of this information within links is displaying atm. Understandable as the same problem seems to be happening in the FHL equivalent of this endpoint. Would you be able to fix this in both if it's not too much trouble? 

This is what is currently displaying for links:  

<img width="638" alt="image" src="https://user-images.githubusercontent.com/106095575/189088908-a2819e95-fd66-4094-ac61-cc74ff9b28e5.png">

Links should be displaying something similar to this though:

<img width="635" alt="image" src="https://user-images.githubusercontent.com/106095575/189089169-a1fb54bb-3b18-4a09-8eb3-208f92ebd069.png">